core/src/integration-test/java/com/scalar/db/api/DistributedStorageAdminIntegrationTestBase.java

```diff
@@ -66,7 +66,7 @@ public abstract class DistributedStorageAdminIntegrationTestBase {
           .addSecondaryIndex(COL_NAME5)
           .addSecondaryIndex(COL_NAME6)
           .build();
-
+  private StorageFactory storageFactory;
   private DistributedStorageAdmin admin;
   private DistributedStorage storage;
```

Comment:
Can you please also get a storage instance in `truncateTable_ShouldTruncateProperly()` similar to the test methods you added/modified in this PR? So we can remove this `storage` field.